Jenkinsfile

```diff
@@ -872,8 +834,8 @@ pass = ${OBS_CREDENTIALS_PASSWORD}
             script {
                 if (env.BRANCH_NAME == 'develop' || env.BRANCH_NAME == 'master') {
                     writeFile(file: 'build.log', text: getBuildLog())
-                    sh 'bin/clean-jenkins-log'
-                    sh 'container-host-files/opt/scf/bin/klog.sh -f ${jobBaseName()}-${BUILD_NUMBER}-scf'
+                    sh "bin/clean-jenkins-log"
+                    sh "container-host-files/opt/scf/bin/klog.sh -f ${jobBaseName()}-${BUILD_NUMBER}-scf"
```

Comment:
This is already on develop. https://github.com/SUSE/scf/blob/develop/Jenkinsfile#L875-L876